src/test/scala/mrpowers/jodie/CDFEnabledSparkSessionTestWrapper.scala

```diff
@@ -0,0 +1,17 @@
+package mrpowers.jodie
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.SparkSession
+
+trait CDFEnabledSparkSessionTestWrapper extends Serializable {
```

Comment:
The only change I see here compared to SparkSessionTestWrapper is `enableChangeDataFeed` configuration. Could we reuse SparkSessionTestWrapper til we need a major change that forced us to create a new session wrapper?

Reply:
makes sense